docs/installation.md

```diff
@@ -104,13 +108,7 @@ pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch
 pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html
 ```
 
-After that, you will need to generate the python code of the thrift config definition. If you changed the thrift later on, you will have to rerun this.
```

Comment:
We will still need this section. Thrift compiler can be installed with anaconda:

`conda install thrift-cpp`